examples/cisco/8000e/8000e-ixia.pb.txt

```diff
@@ -0,0 +1,130 @@
+name: "cisco-ixia"
+nodes: {
+    name: "vxr"
+    vendor: CISCO
+    model: "8201"
+    os: "ios-xr"
+    config: {
+        image: "ios-xr:latest" # please update the image
+        file: "r1.config"
+    }
+    constraints:{
+        key: "memory"
+        value: "20Gi"
+    }
+    services:{
+        key: 22
+        value: {
+            name: "ssh"
+            inside: 22
+        }
+    }
+    services:{
```

Comment:
please use the following outside ports and keys for the g* services:

- gnoi: 9337
- gnmi: 9339
- gribi: 9340
- p4rt: 9559

Reply:
done